### DISCUSSION TITLE
cant run javascript on the webstorex tab

### DISCUSSION THREAD

**Discussion by Go0gl** [3kh0/ext-remover#1160] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
trying to run Extension Launcher for blocklist but the script wont run. I tried it on other tabs and it ran just fine but even on chrome://version/ it wouldnt work.

**Comment by Go0gl** [comment]
I found [https://github.com/3kh0/ext-remover/discussions/845](url) but I cant make my name servers custom as its set by organisation
![Screenshot 2024-09-06 2 39 13 PM](https://github.com/user-attachments/assets/859d537b-0cb3-475b-9b61-62ac4d0ffc79)

**Reply by HelloIloveGimkit2** [reply]
if your school has 2 wifi's and only one of them is forced/managed just use the second one that way you can edit the dns

**Reply by Go0gl** [reply]
Thank you, this allows me to access name servers now but whenever I change the first box to any of the 3 DNS it wont do the popup

**Reply by HelloIloveGimkit2** [reply]
because you have to change all of them

**Reply by HelloIloveGimkit2** [reply]
can't check rn bc I'm not on school wifi.

**Reply by HelloIloveGimkit2** [reply]
also I don't think there's a pop-up review the steps:

pulled from the discussion

In custom name servers, set the first box to 45.90.28.195.
You should probably refresh your connection before proceeding to ensure it is connected.
Visit [this link](https://link-ip.nextdns.io/dca1c2/63e44286cbe90ad0). - This will link your IP address to the DNS server so it knows to rewrite your traffic to the exploit. Only one can be linked at a time, but it can be cached.
Visit this link: https://tiny.cc/bookmarkletexec if you are trying to use Extension Launcher. You may get a Kproxy error message if it's your first time doing this, so try this again after dismissing it.

~~If you want to execute bookmarklets with the side panel, open Google Lens from the context menu.~~

This has been tested and does not work.
Paste your bookmarklet code in the textarea and click Run.

**Reply by HelloIloveGimkit2** [reply]
you should prob have this link: https://link-ip.nextdns.io/dca1c2/63e44286cbe90ad0
copied before hand

**Reply by Go0gl** [reply]
> because you have to change all of them

If I change all of them WiFi wont work and I cant access the links

**Comment by SniperWasTaken** [comment, marked as answer]
If your Chromebook has JavaScript patched, then sadly it won't work.

**Reply by HelloIloveGimkit2** [reply]
are you fucking stupid there's something called bookmarklet executor read the full fucking thing before commenting

**Reply by Go0gl** [reply]
It doesn't seem to not work at all it just doesn't work on any chrome tabs

**Comment by 88411** [comment]
This seems pretty interesting. Is there some other github repo or something where I can see the full instructions. When I try to go to the the discussion that googl it says page not found.

**Reply by HelloIloveGimkit2** [reply]
just refresh the discussion
https://github.com/3kh0/ext-remover/discussions/845

**Reply by HelloIloveGimkit2** [reply]
try that one

**Reply by AshtonDavies** [reply]
> When I try to go to the the discussion that googl it says page not found.

Manually replace /url with /845.

Again, I didn't think it would have any possible use scenarios, but it looks like you guys have found one. Use the instructions if you want, I guess.
